crates/oxc_parser/src/lexer/mod.rs

```diff
@@ -530,7 +536,7 @@ impl<'a> Lexer<'a> {
             return Kind::Eof;
         }
 
-        self.trivia_builder.add_single_line_comment(self.current.token.start, self.offset());
+        self.trivia_builder.add_single_line_comment(self.current.token.start, self.offset(), false);
```

Comment:
@Boshen should this call `add_multi_line_comment` or is the call to `add_single_line_comment` intentional?

Reply:
probably copy pasta error.